### PR TITLE
Build Rails for EL9

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
I get the impression our Jenkins job doesn't understand branching: it can't find altered packages since we're not revbumping. I'd prefer it if we could build a new OS without revbumping. Any thoughts on that?

**Comment by evgeni** [comment]
> I get the impression our Jenkins job doesn't understand branching: it can't find altered packages since we're not revbumping. I'd prefer it if we could build a new OS without revbumping. Any thoughts on that?

Correct. My intent was to run a local `obal release rails_core_packages` which will detect that they are built on el8 but not el9 and only do el9 builds.

File: package_manifest.yaml [hunk 1]
```diff
@@ -65,6 +65,24 @@ copr_projects:
...
+          modules: "{{ core_modules }}"
+          external_repos:
+            - "https://yum.puppet.com/puppet7/el/{{ rhel_8 }}/x86_64/"
+            - "{{ foreman_staging }}/rhel-{{ rhel_8 }}-x86_64"
+          comps_file: "{{ inventory_dir }}/comps/comps-foreman-el{{ rhel_8 }}.xml"
+          buildroot_packages: "{{ core_buildroot_packages }}"
+        - name: "rhel-{{ rhel_9 }}-x86_64"
+          modules: "{{ core_modules }}"
+          external_repos:
+            - "https://yum.puppet.com/puppet7/el/{{ rhel_9 }}/x86_64/"
```
[the remark this answers is not in the document]
Comment (evgeni) [reply package_manifest.yaml:82]: That's a fair point, let me prepare this in another PR and then rebase here.

[the remark this answers is not in the document]
Comment (evgeni) [reply package_manifest.yaml:82]: https://github.com/theforeman/foreman-packaging/pull/10018

File: package_manifest.yaml [hunk 2]
```diff
@@ -66,6 +66,23 @@ copr_projects:
             - "{{ foreman_staging }}/rhel-{{ rhel_8 }}-x86_64"
           comps_file: "{{ inventory_dir }}/comps/comps-foreman-el{{ rhel_8 }}.xml"
           buildroot_packages: "{{ core_buildroot_packages }}"
+    foreman-copr-with-el9:
+      copr_project_name: "foreman-{{ foreman_version }}-staging"
```
Comment (ehelms) [inline package_manifest.yaml:70]: If this does not change, why add a new entry?

Comment (evgeni) [reply package_manifest.yaml:70]: how else would we refer to it when a group is configured to use the `with-el9` config?

**Comment by evgeni** [comment]
Create stuff in Copr:

```
% obal copr-project foreman-copr-with-el9

PLAY [Configure Copr projects] ********************************************************************************************************************************************************************************************************

TASK [copr_project : include_tasks] ***************************************************************************************************************************************************************************************************
included: /home/evgeni/Devel/theforeman/obal/obal/data/roles/copr_project/tasks/create.yml for foreman-copr-with-el9

TASK [copr_project : Create Copr project] *********************************************************************************************************************************************************************************************
changed: [foreman-copr-with-el9]

TASK [copr_project : Configure chroots] ***********************************************************************************************************************************************************************************************
changed: [foreman-copr-with-el9] => (item={'name': 'rhel-8-x86_64', 'modules': ['ruby:2.7', 'python39:3.9', 'nodejs:14', 'postgresql:12'], 'external_repos': ['https://yum.puppet.com/puppet7/el/8/x86_64/', 'https://download.copr.fedorainfracloud.org/results/@theforeman/foreman-nightly-staging/rhel-8-x86_64'], 'comps_file': '/home/evgeni/Devel/theforeman/foreman-packaging/comps/comps-foreman-el8.xml', 'buildroot_packages': ['foreman-build']})
changed: [foreman-copr-with-el9] => (item={'name': 'rhel-9-x86_64', 'external_repos': ['https://yum.puppet.com/puppet7/el/9/x86_64/', 'https://download.copr.fedorainfracloud.org/results/@theforeman/foreman-nightly-staging/rhel-9-x86_64'], 'comps_file': '/home/evgeni/Devel/theforeman/foreman-packaging/comps/comps-foreman-el9.xml', 'buildroot_packages': []})

PLAY RECAP ****************************************************************************************************************************************************************************************************************************
foreman-copr-with-el9      : ok=3    changed=2    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0   
```

Try a release
```
% obal release rubygem-actioncable -e build_package_build_system=copr
…
TASK [build_package : Run build] ******************************************************************************************************************************************************************************************************
ok: [rubygem-actioncable] => (item=rhel-8-x86_64)
changed: [rubygem-actioncable] => (item=rhel-9-x86_64)

TASK [build_package : Created tasks] **************************************************************************************************************************************************************************************************
ok: [rubygem-actioncable] => 
  msg:
  - https://copr.fedorainfracloud.org/coprs/build/6711855
…
TASK [build_package : Watch copr build(s)] ********************************************************************************************************************************************************************************************
ok: [rubygem-actioncable]

PLAY RECAP ****************************************************************************************************************************************************************************************************************************
rubygem-actioncable        : ok=26   changed=3    unreachable=0    failed=0    skipped=21   rescued=0    ignored=0   

```

**Comment by evgeni** [comment]
now running
```
% obal release rails_core_packages -e build_package_build_system=copr
```

**Pull request merged**